src/wallet/coin_selection.rs

```diff
@@ -111,13 +118,26 @@ pub type DefaultCoinSelectionAlgorithm = LargestFirstCoinSelection; // make the
 // prev_txid (32 bytes) + prev_vout (4 bytes) + sequence (4 bytes) + script_len (1 bytes)
 pub(crate) const TXIN_BASE_WEIGHT: usize = (32 + 4 + 4 + 1) * 4;
 
+#[derive(Debug)]
+/// Remaining amount after performing coin selection
+pub enum Excess {
+    /// It's not possible to create spendable output from excess using the current drain output
+    /// NoChange(target, remaining_amount, change_output_fee)
+    NoChange(u64, u64, u64),
```

Comment:
Maybe it would be better to have here and below a struct, instead of a tuple? With 3 elements of the same type, it starts to be difficult to remember which is what
OTOH, structs are more verbose and also you need to find a name for them.

Reply:
What do you think of variants with named fields, like [this](https://users.rust-lang.org/t/enum-field-expressions/2411/6).
It can be a middle ground between both options.
E.g.:
```rust
enum Excess {
      NoChange { dust_threshold: u64, remaining_amount: u64, change_fee: u64 },
      Change { amount: u64, fee: u64 }
}
```

Reply:
That would work great!